optuna/samplers/_tpe/sampler.py

```diff
@@ -119,6 +134,19 @@ def objective(trial):
             for more details.
         seed:
             Seed for random number generator.
+        multivariate:
+            If this is :obj:`True`, the multivariate TPE is used when suggesting parameters.
+            The multivariate TPE is reported to outperform the independent TPE. See `BOHB: Robust
+            and Efficient Hyperparameter Optimization at Scale
+            <http://proceedings.mlr.press/v80/falkner18a.html>`_ for more details.
+
+            .. note::
+                Added in v2.2.0 as an experimental feature. The interface may change in newer
+                versions without prior notice. See
+                https://github.com/optuna/optuna/releases/tag/v2.2.0.
+        warn_independent_sampling:
+            If this is :obj:`True`, a warning message is emitted when
+            the value of a parameter is sampled by using an independent sampler.
```

Comment:
We could clarify mentioning that this argument does nothing if `multivariate` is `False`.